gstools/field/rng.py

```diff
@@ -154,22 +171,22 @@ def gau(self, len_scale, mode_no=1000, **kwargs):
 
         Parameters
         ----------
-            len_scale : float
+            len_scale : :class:`float`
                 the length scale of the distribution
-            mode_no : int, optional
+            mode_no : :class:`int`, optional
                 number of the Fourier modes
             **kwargs
                 not used
 
         Returns
         -------
-            ndarray
+            :class:`numpy.ndarray`
                 the modes
         """
         k = self.create_empty_k(mode_no)
-        for d in range(self.dim):
+        for dim_i in range(self.dim):
             rng = self._get_random_stream()
-            k[d] = rng.normal(0., 1./len_scale, mode_no)
+            k[dim_i] = rng.normal(0., 1./len_scale, mode_no)
         return k
```

Comment:
I think a `d` as a index variable in a short loop is perfectly fine

Reply:
Ok. That's just my handwriting